agent/acs/handler/attach_eni_handler_common.go

```diff
@@ -42,10 +44,26 @@ func (handler *ackTimeoutHandler) handle() {
 	}
 	if !eniAttachment.IsSent() {
 		seelog.Warnf("Timed out waiting for ENI ack; removing ENI attachment record with MAC address: %s", handler.mac)
+		handler.removeENIAttachment(handler.mac)
 		handler.state.RemoveENIAttachment(handler.mac)
 	}
 }
 
+func (handler *ackTimeoutHandler) removeENIAttachment(mac string) {
+	attachmentToRemove, ok := handler.state.ENIByMac(mac)
+	if ok {
+		attachmentId, err := utils.GetENIAttachmentId(attachmentToRemove.AttachmentARN)
+		if err != nil {
+			seelog.Errorf("Failed to get attachment id for %s: %v", attachmentToRemove.AttachmentARN, err)
```

Comment:
error message starts with lower case

Reply:
keeping this consistent with other logs in the file